massa-api/src/private.rs

```diff
@@ -135,36 +147,64 @@ impl MassaRpcServer for API<Private> {
         Ok(w_wallet.get_wallet_address_list())
     }
 
-    async fn node_ban_by_ip(&self, ips: Vec<IpAddr>) -> RpcResult<()> {
-        let network_command_sender = self.0.network_command_sender.clone();
-        network_command_sender
-            .node_ban_by_ips(ips)
-            .await
-            .map_err(|e| ApiError::NetworkError(e).into())
+    async fn node_ban_by_ip(&self, _ips: Vec<IpAddr>) -> RpcResult<()> {
+        // let network_command_sender = self.0.network_command_sender.clone();
```

Comment:
todo here ?

Reply:
Ok. Added. Anyway I will open issue for the API endpoint that will be temporary disabled when this PR is merged